### PR TITLE
Sync Meeting to GoogleCalendar [#149]

### PR DESCRIPTION
**Pull request by hekeemje**

### CODE REVIEW

**Comment by hekeemje** [comment]
image
image

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs [hunk 1]
```diff
@@ -222,5 +225,49 @@ private async Task AddMeetingsFromCalendar(string email, IEnumerable<CalendarVis
                 await _meetingService.AddGoogleCalendarMeetings(item.TeamId, events, userId);
             }
         }
+        private async Task AddMeetingsToCalendar(long? teamId)
+        {
+            var meetings = await _context.Meetings.Where(x => x.TeamId == teamId).ToListAsync();
+
```
Comment (YuraRiabov) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs:229]: I don't think we have to enumerate meetings and check for any

**Review by YuraRiabov — CHANGES_REQUESTED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs (hunk 1 above)
Comment (tatianahutii) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs:229]: I believe we can do this in this situation since we use them later

Comment (YuraRiabov) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs:229]: Ok

**Review by YuraRiabov — APPROVED** [review]
Screenshot_732
LGTM

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**